ledfx/devices/utils/rgbw_conversion.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+import numpy as np
+
+
+class OutputMode(str, Enum):
+    RGB = "RGB"  # RGB data
+    RGBW_None = "RGBW No White"  # No white channel, just set to 0
```

Comment:
Recommend pep8 all caps for enums,, that _None is odd and hard to scan, makes it looks like a keyword.

So RGBW_NONE